output.go

```diff
@@ -60,30 +62,23 @@ type commandOutput struct {
 
 var _ Output = &commandOutput{}
 
+const maxBufferSize = 128 * 1024
```

Comment:
do we still have a hard limit?

Reply:
This is still a hard limit - i'll add an API for configuring next week!

Reply:
i think there are also ways with this library to build effectively (and safely) unbounded buffers. I would prefer that solution, but i think i need to read up on it more